SPECS/mlocate/0001-tests-updatedb-reduce-heirarchy-to-300-from-950.patch

```diff
@@ -0,0 +1,35 @@
+From f9651b4baa2db99fcb98955327a7fc45786fa43f Mon Sep 17 00:00:00 2001
+From: Muhammad Falak R Wani <falakreyaz@gmail.com>
+Date: Fri, 23 Sep 2022 15:35:40 +0530
+Subject: [PATCH] tests: updatedb: reduce heirarchy to 100 from 950
+
+Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>
+---
+ tests/updatedb.at | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/tests/updatedb.at b/tests/updatedb.at
+index fd07466..ac2336f 100644
+--- a/tests/updatedb.at
++++ b/tests/updatedb.at
+@@ -249,7 +249,7 @@ AT_KEYWORDS([updatedb])
+ mkdir d
+ (
+   cd d
+-  for depth in $(seq 1 950); do
++  for depth in $(seq 1 100); do
```

Comment:
The `cd` command for some reasons `segfaults` with a huge `path` string.
100 passes, because the resulting hierarchy is less and the eventual path string is manageable.